docs/content/reference/configuration/aperturectl.md

```diff
@@ -26,17 +26,15 @@ if the `--kube` flag were passed).
 
 ## Format
 
-The aperturectl configuration file uses the following [TOML][] syntax:
+The aperturectl configuration file uses the following [TOML][toml] syntax:
 
 ```toml
 [controller]
 url = "controller hostname:port"
 api_key = "api key for the controller"
 ```
 
-All the fields are required (although the file itself is not). See [Configuring
-aperturectl][] for an example on how to configure aperturectl with [FluxNinja
-Cloud Controller][].
+All the fields are required (although the file itself is not). See [Configuring aperturectl][configure-aperturectl] for an example on how to configure aperturectl with [FluxNinja Cloud Controller][cloud-controller].
```

Comment:
Nit: Why you're reverting the `[]` links? I discovered this feature recently and they're lot easier to read when reading plaintext (as the text doesn't contain repetitions)

Reply:
It is much easier to have reference style links in markdown because change in text doesn't require change in link reference.